src/common/SPELLS/talents/monk.ts

```diff
@@ -12,28 +15,22 @@ export default {
   DIFFUSE_MAGIC_TALENT: { id: 122783, name: 'Diffuse Magic', icon: 'spell_monk_diffusemagic' },
   HEALING_ELIXIR_TALENT: { id: 122281, name: 'Healing Elixir', icon: 'ability_monk_jasmineforcetea' },
   DAMPEN_HARM_TALENT: { id: 122278, name: 'Dampen Harm', icon: 'ability_monk_dampenharm' },
-  // Mistweaver
+  RUSHING_JADE_WIND_TALENT: { id: 116847, name: 'Rushing Jade Wind', icon: 'ability_monk_rushingjadewind', chiCost: 1 },
+
+  //Mistweaver
   MIST_WRAP_TALENT: { id: 197900, name: 'Mist Wrap', icon: 'ability_monk_pathofmists' },
   LIFECYCLES_TALENT: { id: 197915, name: 'Lifecycles', icon: 'ability_monk_souldance' },
   SPIRIT_OF_THE_CRANE_TALENT: { id: 210802, name: 'Spirit of the Crane', icon: 'monk_stance_redcrane' },
-  MANA_TEA_TALENT: { id: 197908, name: 'Mana Tea', icon: 'monk_ability_cherrymanatea', duration: 10000 },
```

Comment:
You removed duration but didn't change the location it was used to not use it (Tier30Comparison)